### PR TITLE
[zh] sync dual-stack ingress-controllers ingress service

### PR DESCRIPTION
**Pull request by asa3311**
sync docs/concepts/services-networking/*

### CODE REVIEW

File: content/zh-cn/docs/concepts/services-networking/ingress-controllers.md [hunk 1]
```diff
@@ -53,12 +53,16 @@ Kubernetes 作为一个项目，目前支持和维护
...
 * [Apache APISIX ingress controller](https://github.com/apache/apisix-ingress-controller) is an [Apache APISIX](https://github.com/apache/apisix)-based ingress controller.
 * [Avi Kubernetes Operator](https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes) provides L4-L7 load-balancing using [VMware NSX Advanced Load Balancer](https://avinetworks.com/).
 -->
 * [AKS 应用程序网关 Ingress 控制器](https://docs.microsoft.com/zh-cn/azure/application-gateway/tutorial-ingress-controller-add-on-existing?toc=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fazure%2Faks%2Ftoc.json&bc=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fazure%2Fbread%2Ftoc.json)
   是一个配置 [Azure 应用程序网关](https://docs.microsoft.com/zh-cn/azure/application-gateway/overview)
   的 Ingress 控制器。
+* [阿里云 MSE Ingress](https://docs.microsoft.com/azure/application-gateway/tutorial-ingress-controller-add-on-existing?toc=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fazure%2Faks%2Ftoc.json&bc=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fazure%2Fbread%2Ftoc.json)
+  是一个 Ingress 控制器，它配置[阿里云原生网关](https://www.alibabacloud.com/help/en/mse/product-overview/cloud-native-gateway-overview?spm=a2c63.p38356.0.0.20563003HJK9is)，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/ingress-controllers.md:64]:
```suggestion
  是一个 Ingress 控制器，它负责配置[阿里云原生网关](https://www.alibabacloud.com/help/en/mse/product-overview/cloud-native-gateway-overview?spm=a2c63.p38356.0.0.20563003HJK9is)，
```

File: content/zh-cn/docs/concepts/services-networking/ingress-controllers.md [hunk 2]
```diff
@@ -53,12 +53,16 @@ Kubernetes 作为一个项目，目前支持和维护
...
+  这也是 [Higress](https://github.com/alibaba/higress) 的商业版本。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/ingress-controllers.md:65]:
```suggestion
  也是 [Higress](https://github.com/alibaba/higress) 的商业版本。
```

File: content/zh-cn/docs/concepts/services-networking/ingress-controllers.md [hunk 3]
```diff
@@ -111,6 +116,8 @@ Kubernetes 作为一个项目，目前支持和维护
   Ingress 控制器，能够提供 API 网关功能。
 * [HAProxy Ingress](https://haproxy-ingress.github.io/) 是一个针对
   [HAProxy](https://www.haproxy.org/#desc) 的 Ingress 控制器。
+* [Higress](https://github.com/alibaba/higress) 是一个基于 [Envoy](https://www.envoyproxy.io) 的API网关，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/ingress-controllers.md:119]:
```suggestion
* [Higress](https://github.com/alibaba/higress) 是一个基于 [Envoy](https://www.envoyproxy.io) 的 API 网关，
```

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 4]
```diff
@@ -732,11 +732,13 @@ Kubernetes Service 类型允许指定你所需要的 Service 类型。
...
-adds to the previous.  This is not strictly required on all cloud providers, but
-the Kubernetes API design for Service requires it anyway.
+adds to the previous. However there is an exception to this nested design. You can
+define a `LoadBalancer` Service by
+[disabling the load balancer `NodePort` allocation.](/docs/concepts/services-networking/service/#load-balancer-nodeport-allocation)
 -->
 服务 API 中的 `type` 字段被设计为层层递进的形式 - 每层都建立在前一层的基础上。
-并不是所有云平台都作如此严格要求，但 Kubernetes 的 Service API 设计要求满足这一逻辑。
+但是，这种层层递进的形式有一个例外。
+你可以通过[禁用负载均衡器的 `NodePort` 分配](/zh-cn/docs/concepts/services-networking/service/#load-balancer-nodeport-allocation)来定义一个 `LoadBalancer` 服务。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/service.md:741]:
```suggestion
你可以在定义 `LoadBalancer` 服务时[禁止负载均衡器分配 `NodePort`](/zh-cn/docs/concepts/services-networking/service/#load-balancer-nodeport-allocation)。
```

**Review by my-git9 — COMMENTED** [review]
/approve
/lgtm
Thanks

**Pull request merged**